py/server/deephaven/parquet.py

```diff
@@ -280,6 +280,90 @@ def write(
         raise DHError(e, "failed to write to parquet data.") from e
 
 
+def write_key_value_partitioned_table(
```

Comment:
Is this method name okay?

Reply:
It is OK because it matches the Java one? Or maybe just `write_partitioned_table` because that's the only/default way to write out a PartitionedTabled?

Reply:
Matching the java name is not the key objective.  It needs to be clear and concise first.

Reply:
The prior name is `write`.  It seems like this should be `write_partitioned`.

Are there multiple ways partitioning can be done?  Is key_value the only way?  If there are mutliple ways, it is likely more pythonic to have function arguments to control the type instead of using the function name.

Reply:
Should this method be separate from `write` or part of `write`?  I may lean to separate, but our python code should be checked for precedent.